openassessment/xblock/static/sass/xb/utilities/_variables.scss

```diff
@@ -114,14 +83,29 @@ $tmg-f3: 0.125s;
 // animation-based settings
 $anim-distance-offviewport: ($baseline-h*100);                      // for larger/page-level animations
 
+//---------------------
+// UXPL colors
+//---------------------
+$uxpl-gray-base: rgb(65, 65, 65);
```

Comment:
I don't think this is possible with the current state of the Pattern Library, but it would have been great to have npm installed the library and then just use the constants from there. We need to work on making that possible.